.github/workflows/ci.yaml

```diff
@@ -0,0 +1,47 @@
+name: CI
+on: [pull_request]
+
+jobs:
+  static-unit:
+    name: Static and Unit tests
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        php-versions: ['7.1', '7.2', '7.3']
+    env:
+      COMPOSER_AUTH: '{"http-basic": {"repo.magento.com": {"username": "021659cd19664f0167a9f727a73f781a","password": "99c7a50a8c29319cafc4574e2ce4f91f"}}}'
```

Comment:
Can we use env variables to set credentials?

Reply:
This will require some ticket looks like, I'll try to figure out if that's doable

Reply:
@BaDos fixed